### ISSUE TITLE
Comments on posts

### ISSUE THREAD

**Issue by aoduor** [ushahidi/platform#296]
Looks like members of the public can't add updates/comments to published posts. I think this would be a valuable feature. (via Nekesa Were)

Aha! Link: https://ushahiditeam.aha.io/features/PROD-747

**Comment by rjmackay** [comment]
cc @jshorland
Any other requirements for comments from BRCK?

**Comment by jshorland** [comment]
Nice to have feature (feedback from quakemap who found comments very valuable): getting notifications when someone comments on a post you've also commented on. Similar to blog commenting notifications.

**Comment by rjmackay** [comment]
@jshorland bumping this to the next sprint

**Comment by sebm90** [comment]
Is this now in Sprint #2 (May 2 - June 3)?

**Comment by rjmackay** [comment]
@sebm90 its probably going to get bumped further out. Is there a solutions project waiting on that??

**Comment by jshorland** [comment]
#508 is related

**Comment by jshorland** [comment]
Closing #508 in favor of this and adding comment here: 

Copying V2 feature request 595:

I don't see a way to limit commenting to logged in users only - only the on/off switch in the site settings. Would be great to add this for both spam protection and accountability. Best would be to assign it as a user role capability that can be selected for assorted user levels.

Nice as an option, but don't want to limit comments to logged-in users on every Ushahidi deployment. Ushahidi would be a rather limited crowd-sourcing tool if the crowd can't comment. BTW, Akismet has done a great job of eliminating spam for us.

**Comment by jshorland** [comment]
This might align nicely with future of "tasks"

**Comment by rowasc** [comment]
https://github.com/ushahidi/platform-client/pull/1229
https://github.com/ushahidi/platform/pull/3207
Noting this 2 PRs from Will that seem to be addressing some of the "comments" functionality, will need to review and test/demo to see where we are exactly on this

**Comment by Erioldoesdesign** [comment]
I would also like to do user testing on this when it goes out. I suspect we should make this 'optional' via admin settings to avoid 'trolling' if that isn't already a specification.

**Comment by rowasc** [comment]
Agree on making it optional. 
I'll let you know when this lands in a test environment and you can run user tests there?

**Comment by Erioldoesdesign** [comment]
That'd be awesome! Usertesting in the test environment is great cause we often gatehr other insights about other features 🙌

**Comment by rowasc** [comment]
Ok. The PR needs work. It's not running, and the migrations do not run, so we will 
- close the PR so it's not there just sitting in our queue without doing anything.
- keep the branches so we can come back to it. 
- keep this issue open and link to the branches.. 
     - https://github.com/ushahidi/platform-client/pull/1229 client work
     - https://github.com/ushahidi/platform/pull/3207 backend work
